src/seedu/addressbook/common/Utils.java

```diff
@@ -13,6 +13,9 @@
      * Returns true if any of the given items are null.
      */
     public static boolean isAnyNull(Object... items) {
+        if (items.length == 0) {
+            return true;
```

Comment:
This is incorrect as an empty list is not equal to a null object. So empty list should assert to False as it does not have any null objects.